src/containers/guest-home-page/login-form/LoginForm.styles.js

```diff
@@ -4,13 +4,27 @@ export const styles = {
     flexDirection: 'column',
     minWidth: { sm: '340px' }
   },
+  loginOptionsContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    height: '40px',
+    padding: '0px 5px',
```

Comment:
better to write padding as `p:'0px 5px',`

Reply:
Thanks. I have made the corrections